dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -892,6 +892,26 @@
 			status = "disabled";
 			label = "QUADSPI";
 		};
+
+		usbotg_fs: usb@40080000 {
```

Comment:
Maybe the definition in `[stm32h743.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/7fe5ab315a4453322780e5655364fe4de2a1b9b6/dts/arm/st/h7/stm32h743.dtsi)` should be removed then?
https://github.com/zephyrproject-rtos/zephyr/blob/7fe5ab315a4453322780e5655364fe4de2a1b9b6/dts/arm/st/h7/stm32h743.dtsi#L22-L34

I have noticed that the interrupts are not in the same order and h743 version has only 3 interrupts.

Reply:
@romainreignier Thanks for raising this.
Actually this change is not correct. Only stm32H742 "and higher" support usbotg_fs. On the other hand usbotg_hs is indeed supported on the whole stm32h7 series.

Reply:
Great catch! I'll move it to stm32h745.dtsi

Reply:
I am still unable to use the sample `samples/subsys/usb/console` on the MachineControl, the USB device is not detected by the kernel on boot.